recipe/set_bld_opts.bat

```diff
@@ -62,7 +62,7 @@ set BLD_OPTS=%WIN64% ^
     NETCDF_LIB=%LIBRARY_LIB%\netcdf.lib ^
     NETCDF_SETTING=yes ^
     NETCDF_HAS_NC4=yes ^
-    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.4" ^
+    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.5" ^
```

Comment:
The fix actually seems pretty obvious.

Reply:
Careful, you are now a windows expert for conda-forge!

Reply:
Oh, god!  Please no!  If I count as a windows expert here we're in very deep trouble!